api/api.yaml

```diff
@@ -161,26 +163,11 @@ paths:
               schema:
                 $ref: '../result/state.yaml#/components/schemas/State.Metrics'
 
-  /results/capabilities:
```

Comment:
Are we getting rid of the capabilities API?  There are references to the capabilities API in `capture/capture.yaml`, `layer1/layer1.yaml`, `port/port.yaml` not to mention `result/capability.yaml`.  If we want to get rid of it, we should do it in all these places & delete result/capability.yaml but do it in a separate PR.

Reply:
Capabilities will be removed and will be done in a separate PR>